src/it/unimi/dsi/fastutil/Function.java

```diff
@@ -40,22 +39,36 @@
  * @see java.util.Map
  */
 
-public interface Function<K,V> {
+public interface Function<K,V> extends java.util.function.Function<K,V> {
+
```

Comment:
I think we should {@inheritDoc} here, and in the following comment explain what the default method does.